operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1280,6 +1284,301 @@ ruler:
   # Set to 'null' to use the Jaeger client's default value
   jaegerReporterMaxQueueSize: 1000
 
+  # -- If set to true, a dedicated query path will be deployed for the ruler and operational mode will be set to use remote evaluation. https://grafana.com/docs/mimir/latest/references/architecture/components/ruler/#remote
+  # -- This is useful for isolating the ruler queries from other queriers (api/grafana).
+  remoteEvaluationDedicatedQueryPath: false
+
+# -- Only deployed if .Values.remoteEvaluation.enabled && .Values.remoteEvaluation.dedicatedQueryPath
```

Comment:
All the comments refer to `.Values.remoteEvaluation.enabled`. Can you replace them with `ruler.remoteEvaluationDedicatedQueryPath`? (`.Values` is an implementation detail)

Reply:
@dimitarvdimitrov - Fixed, thanks :)